src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertransactionsapi/model/MagicLinkRequest.kt

```diff
@@ -6,5 +6,8 @@ import javax.validation.constraints.NotNull
 data class MagicLinkRequest(
   @Schema(description = "The email address to send the magic link to", example = "andrew.barret@company.com")
   @NotNull
-  val email: String
+  val email: String,
+  @Schema(description = "The browser session ID")
+  @NotNull
+  val sessionID: String
```

Comment:
Is the aim to force users to request the magic link and click the magic link within the same session, with the same session ID? Might they request on phone and click on desktop?

Reply:
Thanks, a very good question. Having thought this through I'm now leaning towards a one-time-code instead of a magic link (subject to discussions with the team). We could then insist they enter the one-time-code in the same browser session as they requested it.